libs/components/storybook/src/lib/preview-wrapper/preview-wrapper.component.ts

```diff
@@ -45,40 +45,46 @@ export class PreviewWrapperComponent implements OnInit, OnDestroy {
     }
   }
   public get theme() {
-    return this._theme;
+    return this.#_theme;
   }
 
   public get themeSettings(): SkyThemeSettings {
-    return this._themeSettings;
+    return this.#_themeSettings;
   }
   public set themeSettings(value: SkyThemeSettings) {
-    this._themeSettings = value;
-    if (this.initialized) {
-      this.themeService.setTheme(this._themeSettings);
+    this.#_themeSettings = value;
+    if (this.#initialized) {
+      this.#themeService.setTheme(this.#_themeSettings);
     }
   }
 
-  private _themeSettings = new SkyThemeSettings(
+  #_themeSettings = new SkyThemeSettings(
     SkyTheme.presets.default,
     SkyThemeMode.presets.light
   );
-  private _theme?: 'default' | 'modern-light' | 'modern-dark';
-  private readonly _ngUnsubscribe = new Subscription();
-  private initialized = false;
+  #_theme: 'default' | 'modern-light' | 'modern-dark' = 'default';
+  #ngUnsubscribe = new Subscription();
+  #initialized = false;
+
+  #themeService: SkyThemeService;
+  #renderer: Renderer2;
 
   constructor(
-    private themeService: SkyThemeService,
+    themeService: SkyThemeService,
     @Inject('BODY') private body: HTMLElement,
-    private renderer: Renderer2
-  ) {}
+    renderer: Renderer2
+  ) {
+    this.#themeService = themeService;
+    this.#renderer = renderer;
+  }
 
   public ngOnInit(): void {
-    this.themeService.init(this.body, this.renderer, this.themeSettings);
-    this.initialized = true;
+    this.#themeService.init(this.body, this.#renderer, this.themeSettings);
+    this.#initialized = true;
   }
 
   public ngOnDestroy(): void {
-    this._ngUnsubscribe.unsubscribe();
-    this.themeService.destroy();
+    this.#ngUnsubscribe.unsubscribe();
```

Comment:
Do we need to first run `this.#ngUnsubscribe.next()`? Should we run `complete` instead of `unsubscribe`?

Reply:
Ummm. When I looked deeper we weren't even using that field other than creating it and destroying it. I removed it all together.